### PR TITLE
Zoneout

### PR DESCRIPTION
**Pull request by ASalvail**
Zoneout has become a popular alternative to `dropout` when used in conjunction with recurrent neural networks. This is a wrapper that functions in a similar way to the `DropoutWrapper` of Tensorflow. Since it is not available (as of june 2017) in Tensorflow, this is a code snippet that could benefit anyone using recurrent neural networks.

### CODE REVIEW

File: python/ops/zoneout.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    References
+    ----------
+    [1] Krueger, David, et al. "Zoneout: Regularizing rnns by randomly preserving hidden activations."
+    arXiv preprint arXiv:1606.01305 (2016). https://arxiv.org/abs/1606.01305
+
+    [2] https://github.com/teganmaharaj/zoneout
+    """
+
+    def __init__(self, cell: RNNCell, zoneout_prob: float = 0.15, seed=None):
```
Comment (tboquet) [inline python/ops/zoneout.py]: Typo :).
